### DISCUSSION TITLE
SQLE的openapi如何使用

### DISCUSSION THREAD

**Discussion by alextinng** [actiontech/sqle#476] [Q&A] [answered]
SQLE的openAPI是否有文档？

**Comment by sjjian** [comment, marked as answer]
http://{your address}:10000/swagger/index.html

**Reply by alextinng** [reply]
建议维护到文档上哈~

**Comment by Sharkzeng** [comment]
有使用BASH 脚本抓MYSQL PS库下的语句 通过OPEN API 送给 审核任务的案例没?

**Reply by sjjian** [reply]
open api 的使用使用下面的接口就行，可以自己用工具调试或用脚本调用
![image](https://user-images.githubusercontent.com/18008452/181431978-10dafb18-8e96-4894-9b7c-71674d47a23a.png)
